spec/02-integration/05-proxy/14-upstream-status-header_spec.lua

```diff
@@ -0,0 +1,141 @@
+local helpers = require "spec.helpers"
+local constants = require "kong.constants"
+
+local function setup_db()
+  local bp = helpers.get_db_utils()
+  assert(helpers.dao:run_migrations())
```

Comment:
FYI, the `get_db_utils()` utility will run the migrations for you :) I am proceeding with the merge at this point and will fix it locally, but for next time!